frontend/src/styles.sass

```diff
@@ -1 +1,71 @@
-/* You can add global styles to this file, and also import other style files */
+// /* You can add global styles to this file, and also import other style files */
+
+*
+    margin: 0
+    padding: 0
+    font-family:'Rubik','sans-serif'
+
+//bodies
+body
+   
+    font-size: 14px
+
+.body-light
+    font-weight:300
+
+.body-regular
+    font-weight:400
+
+
+//headers
+h3,h4,h5,h6   
+    font-weight:700
+
+h3
+    font-size:40px
+
+h4
+    font-size:24px
+
+h5
+    font-size:18px
+    font-weight:500
+
+h6
+    font-size:16px
+    font-weight:500
+
+
+//menues
+.menu
+    font-size:14px
+    font-weight:500
+
+
+//tubs
+.tub
+    font-size:16px
+    font-family:600
+
+
+//buttons
+.btn-full
+    background: linear-gradient(90deg, #FD9122 0%, #FD5D34 100%)
+    border-radius: 50px
+
+.btn-ghost
+    border: 1px solid #FF792A
+    box-sizing: border-box
+    border-radius: 50px
+
+//inputs
+.input-dark
+    background: #282F37
+    border-radius: 50px
+
+
+//cards
+
+card
+    background-color: #282F37
+    border-radius: 4px
```

Comment:
Add basic colors separately like this:
![image](https://user-images.githubusercontent.com/24195200/127302481-9e87e75a-6d84-45d8-8c05-1c42c10d9da1.png)

Reply:
I had tried, but after when I built project ,  I saw this error 
*SassError: Undefined variable.
   
13 │   border: 1px solid $border-dark - last word mark underline.
But if i dont use defined color,  only define, project build and start.
I continue finding solution .

Reply:
Should I only define colors ? Or define and use in that style.sass ?

Reply:
Solved